slides/syntroduction/assets/casts/commodore.cast

```diff
@@ -0,0 +1,59 @@
+{"version": 2, "width": 190, "height": 26, "timestamp": 1594732914, "env": {"SHELL": "/bin/bash", "TERM": "xterm-256color"}, "title": "Project Syn - Commodore Demo"}
```

Comment:
Just viewing this on its own, it was too fast for met to grasp what is going on. If this is meant to be accompanied with explanations, this is fine. If it should work on its own, maybe slow it down a bit. Also applies to the other casts.

Reply:
True that, but it's actually a huge amount of work to record a good looking asciinema show. Looking at it on the terminal using `asciinema play` let's you pause it by hitting `space` and on the web by clicking on the pause button. And that's what I intend to do during presentations. If this turns out to be cumbersome, I might fiddle around with the recording file to make it pause longer on some positions.